WcaOnRails/app/views/persons/_personal_records.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="personal-records">
+  <h3 class="text-center"><%= t 'persons.show.personal_records' %></h3>
```

Comment:
Done, see it [here](https://staging.worldcubeassociation.org/persons/2008SEAR01).